tests/mmptest/src/MMPTest.cs

```diff
@@ -711,15 +712,15 @@ public void BuildingSameSolutionTwice_ShouldNotRunACToolTwice ()
 				string project = TI.GenerateUnifiedExecutableProject (test);
 
 				string buildOutput = TI.BuildProject (project);
-				Assert.True (buildOutput.Contains ("actool execution started with arguments"), $"Initial build should run actool");
+				Assert.True (buildOutput.Contains (actool), $"Initial build should run actool");
```

Comment:
Just curious if there is a reason to use the $ before the string if we do not use any variables inside it?

Reply:
no, I did not notice it in the original source (or I would have removed it).